### ISSUE TITLE
Tutorials for TOPP shell and TOPPView

### ISSUE THREAD

**Issue by ghost** [OpenMS/OpenMS-docs#69]
We would like to create tutorials for TOPP shell and TOPPView.

The TOPP shell tutorial will explain how to set an input and manipulate the input data.

The TOPPView tutorial will explain how to visualise the data.

In order to do this we need:
- Some input data to provide to a user so that they can follow the instructions in the tutorial
- A few steps to manipulate this data
- More information about the images in the [current file](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/release/latest/TOPP_tutorial.pdf)
- More information about spectrum and how to interpret a spectrum diagram

@tapaswenipathak feel free to add to this.

**Comment by ghost** [comment]
@tjeerdijk seemed to think that it was a good idea to start with the FileInfo example, so I will start with that. Basically we will have four tutorials for the same action (printing file information to an output file):
- TOPP shell (or the CLI of your choice)
- TOPPView
- pyOpenMS
- Knime

**Comment by tjeerdijk** [comment]
Yes, @greengypsy I indeed think simply reading an mzML file and displaying some statistics (how many spectra, what kind of machine etc) is a good first step. Also, showing how to do this "four ways" addresses the needs of all first-time OpenMS users.

**Comment by ghost** [comment]
@tjeerdijk test
